python/extension/CUDAQuantumExtension.cpp

```diff
@@ -178,6 +178,23 @@ PYBIND11_MODULE(_quakeDialects, m) {
       "ORCA's backends",
       py::arg("input_state"), py::arg("loop_lengths"), py::arg("bs_angles"),
       py::arg("n_samples") = 10000);
+  orcaSubmodule.def(
+      "sample_async",
+      py::overload_cast<std::vector<std::size_t> &, std::vector<std::size_t> &,
+                        std::vector<double> &, std::vector<double> &, int>(
+          &cudaq::orca::sample_async),
+      "Performs Time Bin Interferometer (TBI) boson sampling experiments on "
+      "ORCA's backends",
+      py::arg("input_state"), py::arg("loop_lengths"), py::arg("bs_angles"),
+      py::arg("ps_angles") = nullptr, py::arg("n_samples") = 10000);
```

Comment:
I wonder if the default value of `nullptr` is valid in this case. Do we have a test/example _not_ passing `ps_angles` to this call to make sure it works as intended?

Reply:
The example files (`examples/cpp/providers/orca.cpp` and python equivalent) are _not_ passing `ps_angles`

Reply:
I think the below overload (with no `ps_angles` argument) masked the default value for `ps_angles` as it was probably the overload selected in this case.
Since we have a specific overload for not having the `ps_angles` argument, do we need a default value for it here as that would have no effect (i.e., without providing a value for it, another overload is provided)?